src/main/resources/com/openshift/jenkins/plugins/pipeline/NameValuePair/help-name.html

```diff
@@ -0,0 +1,3 @@
+<div>
```

Comment:
can't tell from the git review tool ... did you link all the help files to the analogous locations for the dsl resources ... they are consumed by the pipeline grooy snippet generator thing